### DISCUSSION TITLE
Support for SSD1283A ?

### DISCUSSION THREAD

**Discussion by ntavares** [Bodmer/TFT_eSPI#1707] [New processor/display support or capability request]
Hi, 
Thank you for this library.

Would it be possible to add support for SSD1283A chip? How could I help (besides testing)? I'm just starting with arduino stuff, so I'm not sure how can I help at a lower level...

For reference, here is a sample:
https://www.aliexpress.com/item/1005001908663162.html

... and datasheets:
http://www.lcdwiki.com/1.6inch_SPI_Module_SSD1283A_SKU:MSP1601

**Comment by Bodmer** [comment]
Yes, it is possible. Cheap screens are mostly left over from previous phone generations, so I expect the shortage will continue. :-(

**Comment by ntavares** [comment]
OK, how can I help? I have one... do you have one? :) No idea where to look at...

**Comment by Onefabis** [comment]
Hi,  is there any perspectives with that screen? I've tried to adopt one of the screen exaples in your library, editing init sequence, but without any luck. Definitelly missed something.
